src/utils.js

```diff
@@ -144,10 +182,20 @@ const createNewPeerInfo = () =>
     });
   });
 
+/**
+ * @desc convert String To Uint8Array
+ * @param {string} string
+ * @return {Uint8Array}
+ */
+const convertStringToUint8Array = string =>
```

Comment:
there is no test for this method and it's `UInt` not `Uint`

Reply:
I have removed this method because of we can use  Buffer.from instead